db/re/mob_skill_db.txt

```diff
@@ -12322,3 +12441,18 @@
 3633,EP16_2_VENOM_KIMERA@NPC_WIDESIGHT,attack,669,1,10000,0,30000,yes,self,always,0,,,,,,,
 3633,EP16_2_VENOM_KIMERA@NPC_ACIDBREATH,attack,657,6,10000,3000,25000,no,target,always,0,,,,,,,
 3633,EP16_2_VENOM_KIMERA@RG_STRIPARMOR,attack,217,5,500,1000,5000,no,target,always,0,,,,,,,
+
+3741,MECHASPIDER@AL_TELEPORT,idle,26,1,10000,0,0,yes,self,rudeattacked,,,,,,,,
+3741,MECHASPIDER@AL_TELEPORT,idle,26,1,5000,0,5000,yes,self,rudeattacked,,,,,,,,
+3741,MECHASPIDER@AL_TELEPORT,walk,26,1,5000,0,5000,yes,self,rudeattacked,,,,,,,,
+3741,MECHASPIDER@NPC_CALLSLAVE,attack,352,1,10000,0,30000,yes,self,always,0,,,,,,,
+3741,MECHASPIDER@NPC_CALLSLAVE,idle,352,1,10000,0,30000,yes,self,always,0,,,,,,,
+3741,MECHASPIDER@NPC_WIDESTONE,chase,666,5,10000,500,20000,no,self,myhpltmaxrate,80,,,,,,,
+3741,MECHASPIDER@NPC_WIDESTONE,attack,666,5,10000,500,20000,no,self,myhpltmaxrate,80,,,,,,,
+3741,MECHASPIDER@NPC_GUIDEDATTACK,chase,172,5,500,0,20000,no,target,always,0,,,,,,,
+3741,MECHASPIDER@NPC_LICK,attack,206,5,2000,0,5000,yes,target,always,0,,,,,,,
+3741,MECHASPIDER@NPC_PETRIFYATTACK,chase,180,5,2000,500,5000,no,target,always,0,,,,,,,
+3741,MECHASPIDER@NPC_SUMMONSLAVE,attack,196,1,10000,700,10000,no,self,slavele,3,3736,,,,,,
+3741,MECHASPIDER@NPC_SUMMONSLAVE,idle,196,1,10000,700,10000,no,self,slavele,3,3736,,,,,,
+3741,MECHASPIDER@AL_HEAL,idle,28,10,10000,0,5000,yes,self,mystatuson,hiding,,,,,,,
+3741,MECHASPIDER@AL_HEAL,idle,28,10,10000,0,10000,yes,self,myhpltmaxrate,50,,,,,,,
```

Comment:
Looks like this change was meant to be part of https://github.com/Latiosu/chocobotro/pull/8

Reply:
Actually never mind, it makes sense for it to be here 👍